region/norfair/east/Lava Dive Room.json

```diff
@@ -580,7 +580,6 @@
           "canSuitlessLavaDive"
         ]},
         "canSpringwall",
```

Comment:
Maybe something to account for the extra lava damage as well?
```suggestion
        "h_HeatedSpringwall",
```

Reply:
- Each of these lava room strats are going to have unique frame counts to try again
- I think this springwall is easier if you have the general idea to the trick, because the movements are all slower.
- This was made notable for having similar requirements to "lava dive with HiJump" and I wanted that to be notable.